elastic/elastic-stack-testing
# Allow monitoring cleanup

Allows monitoring doc parity steps with `DELETE` to run. Also allows override of ES_BUILD_URL as per readme.

To test run:

```
ES_BUILD_URL=https://snapshots.elastic.co/8.1.0-7004acda AIT_STACK_PRODUCT=kibana ./playbooks/monitoring/buildenv.sh
```

Note that you might still get this failure:

```
TASK [xpack_metricbeat : Wait for log file and search string to be present] ****
fatal: [aithost]: FAILED! => {
    "changed": false,
    "elapsed": 60
}

MSG:

Timeout when waiting for search string (2022-01-12T06:09|2022-01-12T06:08|2022-01-12T06:10)([\w*\W*\n]*).*Connection to backoff\(elasticsearch.*established in /home/vagrant/metricbeat-8.1.0-SNAPSHOT-linux-x86_64/logs/metricbeat-20220112.ndjson
```

Trying to get that working now.
